### DISCUSSION TITLE
Does minio pvc stable and reliable？We want use it in production.

### DISCUSSION THREAD

**Discussion by WyattJia** [minio/minio#13042] [Q&A] [answered]
We have 1000-2000 images per a second, 20000000 images per a day to store on our kubernetes cluster. We want use minio pvc to store those pictures, but we don’t know if this plan is feasible. Can anyone share your case of using minio pvc to store large-scale data in a production environment? And we still don’t know if kubernetes pvc technology is trustworthy. Thank you very much.

**Comment by harshavardhana** [comment, marked as answer]
> We have 1000-2000 images per a second, 20000000 images per a day to store on our kubernetes cluster. We want use minio pvc to store those pictures, but we don’t know if this plan is feasible. Can anyone share your case of using minio pvc to store large-scale data in a production environment? And we still don’t know if kubernetes pvc technology is trustworthy. Thank you very much.

You are better off getting a subscription with us https://min.io/pricing we can help your architecture your deployment at scale.

PVC's have been stable for years.

**Reply by WyattJia** [reply]
Can you provide services in China? We use minio in the internal network of our self-built machine room.

**Reply by WyattJia** [reply]
And can you share your case of using minio pvc to store large-scale data in a production environment?
